### DISCUSSION TITLE
A cheat sheet to migrate from Moq to NSubstitute

### DISCUSSION THREAD

**Discussion by giscus** [timdeschryver/timdeschryver.dev#270] [Announcements]
# A cheat sheet to migrate from Moq to NSubstitute

A quick reference to help the migration from Moq to NSubstitute.

https://timdeschryver.dev/blog/a-cheat-sheet-to-migrate-from-moq-to-nsubstitute

**Comment by JasonBock** [comment]
FWIW I've written a mocking framework that's based on the source generator feature in C#. It's called Rocks - feel free to check it out and let me know what you think.

https://github.com/JasonBock/Rocks/
https://nuget.org/packages/Rocks
